.gitignore

```diff
@@ -13,3 +13,6 @@
 
 # Log files
 *.log
+
+tools/psoxy-test/node_modules
+tools/psoxy-test/package-lock.json
```

Comment:
FYI, `.gitignore`s are hierarchical; so you can put this in a `.gitignore` within `tools/psoxy-test/`, where it will be robust against directory being renamed/moved.